camelot/view/art.py

```diff
@@ -221,11 +214,25 @@ def getQPixmap(self):
         engine = FontIconEngine()
         engine.font_family = 'Font Awesome 5 Free'
         engine.code = chr(int(FontIcon._name_to_code[self.name], 16))
-        engine.color = self._color
+        engine.color = QtGui.QColor(self.color)
 
         return engine.pixmap(QtCore.QSize(self.pixmap_size, self.pixmap_size), QtGui.QIcon.Normal, QtGui.QIcon.Off)
 
 
+def from_admin_icon(admin_icon):
+    """Convert :class:`camelot.admin.icon.Icon` object to :class:`camelot.view.art.Icon` or
+    :class:`camelot.view.art.FontIcon`.
+
+    If the name of the admin icon ends with .png, a :class:`camelot.view.art.Icon` object
+    will be returned with the module parameter set to vfinance.
+    """
+    if admin_icon.name.lower().endswith('.png'):
+        import vfinance
+        return Icon(admin_icon.name, vfinance)
```

Comment:
Hier moet mischien nog een betere oplossing voor gevonden worden.

Voor de *.png image based iconen moeten we ook de module weten waar die resource in zit. Dit hoort niet echt thuis in de admin Icon class vermits dit specifiek is voor python. Dit zijn enkel nog de venice, multivers en nbb icons. Moet ik die import vfinance in camelot wegwerken door bijvoorbeeld een set_icon_resource_module(module) functie toe te voegen die we dan in vfinance aanroepen? Andere suggesties?

Reply:
neen, dit mag zeker niet :)

suggestie zou zijn om er dan een qt resource van te maken, en die resources bij in de launcher te steken ??

uiteindelijk moeten we toch ofwel de 'images' zelf in de ui kant steken, of de model kant moet de inhoud aanleveren,
maar geen hybride.

dus ik denk dat we de keuze hebben tss in de resources steken, ofwel in de Icon zelf de mogelijkheid te geven om de bitmap mee te sturen ... en dan denk ik dat de resources het meest eenvoudige/snelste gaat zijn ?

Reply:
Ik heb de icoontjes als resources mee in de launcher geplaatst.

Dit wil wel zeggen dat deze niet beschikbaar zijn als je de unit tests draait zonder de launcher. Is dit een issue?